src/app/components/reminder.js

```diff
@@ -5,31 +5,46 @@ import { notes, messages } from './../constants/constants';
 export default class Reminder {
     constructor(container) {
         this.container = container;
+        this.form = this.container.querySelector('.health-support__form');
         this.apiUrl = this.container.getAttribute('data-apiUrl');
+        this.selectors = [...this.container.querySelectorAll('select')];
         this.day = this.container.querySelector('.js-selectDay');
         this.time = this.container.querySelector('.js-selectTime');
         this.setBtn = this.container.querySelector('.js-setReminder');
         this.setBtn.disabled = true;
-        this.selected_day = this.day.value;
-        this.selected_time = this.time.value;
+        this.selectedDay = this.day.value;
+        this.selectedTime = this.time.value;
+        this.selectorsValue;
 
-        this.day.addEventListener('change', this.selectRemindTime);
-        this.time.addEventListener('change', this.selectRemindTime);
+        this.form.addEventListener('change', this.validate);
         this.setBtn.addEventListener('click', this.setReminder);
     }
 
-    selectRemindTime = (event) => {
-        const selectEl = event.target;
+    validate = (event) => {
```

Comment:
Here you are reassigning the value of `selectorsValue` and then in `disableBtn` checking the value of last `select`. So if you choose first the time of the reminder, you can submit the reminder,

Try to use `filter` method to return `invalid` fields and then check if you have any `invalid` ones and enable the submit `button`. And you don't have to set `value="0"` in first `option` in `select`, just use `value` without a value.  Also check what this means [`!!value`](https://stackoverflow.com/questions/29312123/how-does-the-double-exclamation-work-in-javascript), it's helpful if you want to convert something into `boolean`